pineTreeQuiz.html

```diff
@@ -0,0 +1,32 @@
+<!--kffkmg-->
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="pineTreeQuiz.css">
+</head>
+<body>
+	<header>
+		<h1>Make your own Pine Tree!</h1>
+	</header>
+
+	<div>
+		<form action="#" onsubmit="treeFunction()">
```

Comment:
Avoid using js in your html. Try to accomplish this instead with event handlers.